R/convertHistoricalData.R

```diff
@@ -12,17 +12,26 @@
 #' @importFrom stats aggregate
 #' @examples
 #' \dontrun{
-#'data <- convertHistoricalData(
+#' data <- convertHistoricalData(
 #'   mif = "path/to/historical.mif",
 #'   project = "NAVIGATE",
 #'   regionMapping = "path/to/region_mapping_NAVIGATE.csv"
-#')
-#'}
+#' )
+#' }
 #' @export
 convertHistoricalData <- function(mif, project, regionMapping = NULL) {
   hist <- suppressWarnings(as.quitte(mif))
 
-  varmap <- getTemplate(project) %>%
+  if (length(project) > 1) {
+    m <- NULL
+    for (i in project) {
+      m <- rbind(m, getTemplate(i))
+    }
+  } else {
```

Comment:
do you actually need this `if`. I mean, if `length(project) == 1`, then the `for` loop would just run over one item and work as well…

Reply:
done